scripts/check-version.sh

```diff
@@ -6,20 +6,11 @@ set -e
 function check_versions_consistent () {
   local PACKAGE_VERSION=$(yq eval '.version' ./package.json)
   local PACKAGE_LOCK_VERSION=$(yq eval '.version' ./package-lock.json)
```

Comment:
I think you'll need the `-o yaml` option here in order for this to work now as well as on line 43?